packages/api/tests/lib/test-granule-delete.js

```diff
@@ -290,53 +290,6 @@ test.serial('deleteGranuleAndFiles() succeeds if a file is not present in S3', a
   );
 });
 
-test.serial('deleteGranuleAndFiles() will not delete a granule or its S3 files if the PG file delete fails', async (t) => {
```

Comment:
This test is no longer necessary since deleting files from PG is no longer handled by `deleteGranuleAndFiles()` but happens automatically when a granule record is removed from PG

Reply:
Nice work!

Reply:
One catch - I think we need to update api/lib/granule-delete as well, as the individual calls to delete files is no longer called for.